3_visualize/src/plot_data.R

```diff
@@ -0,0 +1,45 @@
+plot_data <- function(plot_out = = "3_visualize/out/figure_1.png", eval_data){
```

Comment:
It is best to pass all filepaths in when you use the function, so that it is obvious on the top-level, where different outputs are located and what they are called. I would recommend not setting a default for `plot_out` and passing in the string `"3_visualize/out/figure_1.png"` when the function is called.

Reply:
change made